frontend/lib/verification/scanner/verification_qr_content_parser.dart

```diff
@@ -1,53 +0,0 @@
-import 'dart:convert';
```

Comment:
This file was moved to `identification/verification_workflow/verification_qr_content_parser.dart`.